README.md

```diff
@@ -1 +1 @@
-# learn-react
\ No newline at end of file
+# learn-react 
```

Comment:
This is great, the only thing now is that this file has a trailing space, if you remove that we can merge